spec/openapi.yaml

```diff
@@ -6468,6 +6468,84 @@ components:
             The subject field is used to denote subject of the MMS message and
             has a maximum size of 64 characters long
           example: New Products
+    DetailMessage:
+      title: Message
+      required:
+        - content
+        - destination_number
+      type: object
+      properties:
```

Comment:
Can we please order these to match a real response too please? If it makes reviewing easier, it will be more recognisable when a user makes a request and sees the same (or very close) ordering in the real response as they saw in the docs! Thank you :)

Reply:
Much better @SamRussellMM